validation/src/test/scala/hmda/validation/rules/lar/validity/V220Spec.scala

```diff
@@ -13,4 +15,18 @@ class V220Spec extends PropSpec with PropertyChecks with MustMatchers with LarGe
       }
     }
   }
+
+  val badLoanTypeGen: Gen[Int] = {
+    val belowRange = Gen.choose(Integer.MIN_VALUE, 0)
+    val aboveRange = Gen.choose(5, Integer.MAX_VALUE)
+    Gen.oneOf(belowRange, aboveRange)
+  }
+
+  property("Loan Type other than 1,2,3,4 is invalid") {
+    forAll(larGen, badLoanTypeGen) { (lar: LoanApplicationRegister, x: Int) =>
+      val invalidLoan: Loan = lar.loan.copy(loanType = x)
+      V220(invalidLoan) mustBe Failure("is not contained in valid values domain")
```

Comment:
This message does not match the "Loan type is missing or is not in range 1-4" that I see in one the 2016 edit document.  I'm guessing that is OK for now, but I'd like to ask explicitly: is that OK for now?  Do we have a plan for making the correct user-facing error messages at some point?

I ask partly because, in the absence of other constraints, it seems easier to implement a rule and its failure message together.  (But there may well be reasons for doing it in a different way.  For example: maybe the 2017 messages won't necessarily match the 2016 ones.  I don't know at the moment, but now is a good time to find out.)